### PR TITLE
[Remove Spring extender usages] LPS-159168 commerce-currency-service | Part 1

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @dantewang @tinatian,

Resend. 

This PR is to import default company value during the CommerceCurrencyService upgrade process, and prepare for removing spring extender usage in commerce-currency-service, thanks!

https://issues.liferay.com/browse/LPS-159168
Previous PR: https://github.com/liferay-core-infra/liferay-portal/pull/1343

### CODE REVIEW

**Comment by JanisZhang** [comment]
ci:reopen

**Comment by JanisZhang** [comment]
@zhangyan0701
Hi Summer, Could you please help to scan the test result? Thank you!

**Review by dantewang — CHANGES_REQUESTED** [review]
@JanisZhang Please see my comments. I'm going to close this PR and let's have a talk later.

File: modules/apps/commerce/commerce-currency-service/src/main/java/com/liferay/commerce/currency/internal/upgrade/registry/CommerceCurrencyImportDefaultValueProcess.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+/**
+ * @author Janis Zhang
+ */
+public class CommerceCurrencyImportDefaultValueProcess extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
```
[the remark this answers is not in the document]
Comment (JanisZhang) [reply modules/apps/commerce/commerce-currency-service/src/main/java/com/liferay/commerce/currency/internal/upgrade/registry/CommerceCurrencyImportDefaultValueProcess.java:78]: @dantewang Ok, thanks.

File: modules/apps/commerce/commerce-currency-service/src/main/java/com/liferay/commerce/currency/internal/model/listener/PortalInstanceLifecycleListenerImpl.java [hunk 2]
```diff
@@ -1,51 +0,0 @@
...
-import com.liferay.commerce.currency.internal.model.listener.util.ImportDefaultValuesUtil;
-import com.liferay.commerce.currency.model.CommerceCurrency;
-import com.liferay.commerce.currency.service.CommerceCurrencyLocalService;
-import com.liferay.portal.instance.lifecycle.BasePortalInstanceLifecycleListener;
-import com.liferay.portal.kernel.model.Company;
-
-/**
- * @author Marco Leo
- */
-public class PortalInstanceLifecycleListenerImpl
```
[the remark this answers is not in the document]
Comment (JanisZhang) [reply modules/apps/commerce/commerce-currency-service/src/main/java/com/liferay/commerce/currency/internal/model/listener/PortalInstanceLifecycleListenerImpl.java:26]: Got it.

**Pull request closed without merging**